src/test/java/filter_post/FilterPostTest.java

```diff
@@ -191,7 +191,7 @@ public void testDsGatewayCatchIOException() {
     @Test
     public void testDsGatewayCatchCsvException() {
         // The csv file contains a row that does not have the same number of columns as the header row.
-        String path = "src/test/java/filter_post/error.csv";
```

Comment:
This was supposed to be intentional. The `error.csv` file was designed so that a `CsvException would be thrown. Although this test still passes, it does not test the specified behaviour.

Reply:
So sorry I think I meant to change that in my tests but then had your test open and and accidentally changed that there!